HW_1/task_3.py

```diff
@@ -0,0 +1,16 @@
+# Напишите программу, которая принимает на вход координаты точки (X и Y) и выдаёт номер четверти плоскости, в которой находится эта точка (или на какой оси она находится).
+
+# Ввод: два значения типа <int>
+# Вывод: значение типа <int> либо значение типа <str>
+
+x_coord = int(input('Введите координату x: '))
+y_coord = int(input('Введите координату y: '))
+
+if x_coord > 0 and y_coord > 0:

```

Comment:
Условия можно свернуть относительно нуля.
```suggestion
if x_coord > 0 < y_coord:

```